kernelci.org/content/en/docs/tests/howto_external.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "How-To-External"
+date: 2023-02-08
+description: "How to add a new external test suite"
+weight: 1
+---
+
+The section contains the steps to add an individual test to other [CI Services in production](https://github.com/orgs/kernelci/projects/16).
+
+The fastest way to add the test is to use existing test suites:
+
+## LTP
+
+* LTP Test source lives at the [Linux Test Project](https://linux-test-project.github.io/)
+* Contact: Send questions to the mailing list **ltp@lists.linux.it**. For full [contact information](https://github.com/linux-test-project/ltp/wiki/Contact-Info), including IRC, see the [LTP wiki](https://github.com/linux-test-project/ltp/wiki/Developers)
+* Proposing patches: You can submit Pull Requests to the [LTP project on GitHub](https://github.com/linux-test-project/ltp).
+* Documentation: The [LTP wiki](https://github.com/linux-test-project/ltp/wiki/Developers) has plenty of documentation to get you started.  You have a [step-by-step tutorial](https://github.com/linux-test-project/ltp/blob/master/doc/c-test-tutorial-simple.txt) to create a new C test, [Guidelines](https://github.com/linux-test-project/ltp/blob/master/doc/test-writing-guidelines.txt), and other documentation.
+
+## kselftests
+
+* The kernel contains a set of “self tests” under the tools/testing/selftests/
+directory
+* See [Contributing to kselftests](https://www.kernel.org/doc/html/latest/dev-tools/kselftest.html#contributing-new-tests) for instructions to add tests
+* You can find more details at the [kselftests wiki](https://kselftest.wiki.kernel.org)
+* Contact: Send questions to the mailing list **linux-kselftest@vger.kernel.org** or join **#linux-kselftest** IRC Channel on freenode
+
+If neither of those testsuites work, you can announce your test [TODO
```

Comment:
What kind of mailing list would that be? Maybe the `automated-testing` one?

Reply:
Works for me.